lib/signs_ui/signs/sign.ex

```diff
@@ -10,4 +10,9 @@ defmodule SignsUI.Signs.Sign do
       enabled?: Map.get(values, "enabled")
     }
   end
+
+  def update_enabled(sign, enabled_string) do
+    enabled? = enabled_string == "true"
```

Comment:
I generally think of this as the responsibility of the controller. That's the boundary where external input is validated and converted into the stuff that the app uses internally. Passing around strings like `"true"` is kind of a code smell, to me.